cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -49,31 +49,28 @@ if node['cdo-apps']['daemon']
   if node.chef_environment == 'staging' && node.name == 'staging'
     cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
     cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'import_google_sheets')
-    cronjob at:'*/2 * * * *', do:deploy_dir('bin','run_server_generate_pdfs'), notify:'brendan@code.org'
-    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','run_server_generate_curriculum_pdfs'), notify:'brian@code.org'
-    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs'), notify:'brian@code.org'
-    cronjob at:'*/2 * * * *', do:dashboard_dir('bin','build_scripts'), notify:'dev@code.org'
-    cronjob at:'*/5 * * * *', do:deploy_dir('bin','fetch-external-resources'), notify:'dev@code.org'
+    cronjob at:'*/2 * * * *', do:deploy_dir('bin','run_server_generate_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','run_server_generate_curriculum_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs'), notify:'dev+crontab@code.org'
+    cronjob at:'*/2 * * * *', do:dashboard_dir('bin','build_scripts'), notify:'dev+crontab@code.org'
+    cronjob at:'*/5 * * * *', do:deploy_dir('bin','fetch-external-resources'), notify:'dev+crontab@code.org'
   end
 
   if node.chef_environment == 'production'
-    cronjob at:'*/15 * * * *', do:deploy_dir('bin','activity-monitor')
-    cronjob at:'@daily', do:deploy_dir('bin', 'send_workshop_reminder_emails')
-    cronjob at:'0 16 * * *', do:dashboard_dir('bin','scheduled_ops_emails')
+    cronjob at:'5 6 * * *', do:deploy_dir('bin', 'send_workshop_reminder_emails')
+    cronjob at:'15 16 * * *', do:dashboard_dir('bin','scheduled_ops_emails')
     cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'index-users-in-solr')
-    cronjob at:'@daily', do:deploy_dir('bin', 'update-hoc-map')
+    cronjob at:'25 7 * * *', do:deploy_dir('bin', 'update-hoc-map')
   end
 
+  # all environments
   cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'process_forms')
-  cronjob at:'0 * * * *', do:deploy_dir('bin', 'analyze_hoc_activity')
+  cronjob at:'35 * * * *', do:deploy_dir('bin', 'analyze_hoc_activity')
   cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'deliver_poste_messages')
   cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'geocode_hoc_activity')
-  cronjob at:'@daily', do:deploy_dir('bin', 'cron', 'ops_data_pull')
+  cronjob at:'55 8 * * *', do:deploy_dir('bin', 'cron', 'ops_data_pull')
 end
 
-# Restart at ~02:00am PST (~ 9am UTC) as workaround for Rails 4.2 memory leak.
-# We will no longer need this as soon as we complete the Ruby 2.2 upgrade.
-cronjob at:"#{rand(20)} 9 * * *", do:'service dashboard upgrade'
 cronjob at:'45 * * * *', do:"#{deploy_dir('bin','upload-logs-to-s3')} dashboard pegasus"
```

Comment:
While you're in the neighborhood, would you mind randomizing when the upload to s3 occurs across the frontends?

You can do this by replacing 45 with #{rand(60)}.   Each frontend will pick a different random minute in the range 0..59.